lib/utils/middleware.js

```diff
@@ -0,0 +1,16 @@
+export default function init(entry, options) {
+    var handlers = [];
+
+    return {
```

Comment:
Me toi, discovered it with https://egghead.io/lessons/javascript-redux-simplifying-the-arrow-functions